app/service/group_member_service.rb

```diff
@@ -0,0 +1,46 @@
+class GroupMemberService
+  Row = Struct.new(:name, :email, :role, :actions, :membership)
+
+  attr_reader :group, :current_user
+
+  class << self
+    def call(**args)
+      new(**args)
+    end
+  end
+
+  def initialize(group:, current_user:)
+    @group = group
+    @current_user = current_user
+  end
+
+  def show_actions?
```

Comment:
Should we just check if there are any actions that have been added to the rows, rather than the user's role? If we've added an action in our row setup, we'd definitely want to `show_actions?` anyway

Reply:
I thought this originally but I think we need to show an empty actions column when the current_user is a group admin and there are no actions. This happens when there are no editors in the group.

Reply:
Cool, makes sense then!

Reply:
wait! I checked with the designers and they said don't show the column if there are no actions, so you were right.

I've updated the code and tests.